[2023-04-15T14:13:41.111+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: joke_collector.joke_collector_task scheduled__2023-04-11T15:00:00+00:00 [queued]>
[2023-04-15T14:13:41.124+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: joke_collector.joke_collector_task scheduled__2023-04-11T15:00:00+00:00 [queued]>
[2023-04-15T14:13:41.124+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-15T14:13:41.125+0000] {taskinstance.py:1289} INFO - Starting attempt 2 of 2
[2023-04-15T14:13:41.125+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-15T14:13:41.142+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonVirtualenvOperator): joke_collector_task> on 2023-04-11 15:00:00+00:00
[2023-04-15T14:13:41.146+0000] {standard_task_runner.py:55} INFO - Started process 88 to run task
[2023-04-15T14:13:41.149+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'joke_collector', 'joke_collector_task', 'scheduled__2023-04-11T15:00:00+00:00', '--job-id', '129', '--raw', '--subdir', 'DAGS_FOLDER/dag_joke_collector.py', '--cfg-path', '/tmp/tmprtvd3x0e']
[2023-04-15T14:13:41.150+0000] {standard_task_runner.py:83} INFO - Job 129: Subtask joke_collector_task
[2023-04-15T14:13:41.231+0000] {task_command.py:389} INFO - Running <TaskInstance: joke_collector.joke_collector_task scheduled__2023-04-11T15:00:00+00:00 [running]> on host 05adb9f68419
[2023-04-15T14:13:41.332+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=joke_collector
AIRFLOW_CTX_TASK_ID=joke_collector_task
AIRFLOW_CTX_EXECUTION_DATE=2023-04-11T15:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-11T15:00:00+00:00
[2023-04-15T14:13:41.334+0000] {process_utils.py:179} INFO - Executing cmd: /usr/local/bin/python -m virtualenv /tmp/venvzleimtqv --system-site-packages
[2023-04-15T14:13:41.342+0000] {process_utils.py:183} INFO - Output:
[2023-04-15T14:13:43.214+0000] {process_utils.py:187} INFO - created virtual environment CPython3.7.16.final.0-64 in 956ms
[2023-04-15T14:13:43.215+0000] {process_utils.py:187} INFO -   creator CPython3Posix(dest=/tmp/venvzleimtqv, clear=False, no_vcs_ignore=False, global=True)
[2023-04-15T14:13:43.216+0000] {process_utils.py:187} INFO -   seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/***/.local/share/virtualenv)
[2023-04-15T14:13:43.216+0000] {process_utils.py:187} INFO -     added seed packages: pip==23.0.1, setuptools==67.4.0, wheel==0.38.4
[2023-04-15T14:13:43.217+0000] {process_utils.py:187} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2023-04-15T14:13:43.266+0000] {process_utils.py:179} INFO - Executing cmd: /tmp/venvzleimtqv/bin/pip install -r /tmp/venvzleimtqv/requirements.txt
[2023-04-15T14:13:43.274+0000] {process_utils.py:183} INFO - Output:
[2023-04-15T14:13:47.419+0000] {process_utils.py:187} INFO - Collecting requests==2.25.1
[2023-04-15T14:13:47.687+0000] {process_utils.py:187} INFO -   Downloading requests-2.25.1-py2.py3-none-any.whl (61 kB)
[2023-04-15T14:13:47.733+0000] {process_utils.py:187} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 61.2/61.2 kB 1.3 MB/s eta 0:00:00
[2023-04-15T14:13:47.777+0000] {process_utils.py:187} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.7/site-packages (from requests==2.25.1->-r /tmp/venvzleimtqv/requirements.txt (line 1)) (2022.12.7)
[2023-04-15T14:13:47.782+0000] {process_utils.py:187} INFO - Requirement already satisfied: urllib3<1.27,>=1.21.1 in /home/***/.local/lib/python3.7/site-packages (from requests==2.25.1->-r /tmp/venvzleimtqv/requirements.txt (line 1)) (1.26.15)
[2023-04-15T14:13:47.785+0000] {process_utils.py:187} INFO - Requirement already satisfied: chardet<5,>=3.0.2 in /home/***/.local/lib/python3.7/site-packages (from requests==2.25.1->-r /tmp/venvzleimtqv/requirements.txt (line 1)) (4.0.0)
[2023-04-15T14:13:47.880+0000] {process_utils.py:187} INFO - Collecting idna<3,>=2.5
[2023-04-15T14:13:47.922+0000] {process_utils.py:187} INFO -   Downloading idna-2.10-py2.py3-none-any.whl (58 kB)
[2023-04-15T14:13:47.946+0000] {process_utils.py:187} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 58.8/58.8 kB 2.8 MB/s eta 0:00:00
[2023-04-15T14:14:06.767+0000] {process_utils.py:187} INFO - Installing collected packages: idna, requests
[2023-04-15T14:14:06.768+0000] {process_utils.py:187} INFO -   Attempting uninstall: idna
[2023-04-15T14:14:06.771+0000] {process_utils.py:187} INFO -     Found existing installation: idna 3.4
[2023-04-15T14:14:06.772+0000] {process_utils.py:187} INFO -     Not uninstalling idna at /home/***/.local/lib/python3.7/site-packages, outside environment /tmp/venvzleimtqv
[2023-04-15T14:14:06.774+0000] {process_utils.py:187} INFO -     Can't uninstall 'idna'. No files were found to uninstall.
[2023-04-15T14:14:06.882+0000] {process_utils.py:187} INFO -   Attempting uninstall: requests
[2023-04-15T14:14:06.884+0000] {process_utils.py:187} INFO -     Found existing installation: requests 2.28.2
[2023-04-15T14:14:06.884+0000] {process_utils.py:187} INFO -     Not uninstalling requests at /home/***/.local/lib/python3.7/site-packages, outside environment /tmp/venvzleimtqv
[2023-04-15T14:14:06.885+0000] {process_utils.py:187} INFO -     Can't uninstall 'requests'. No files were found to uninstall.
[2023-04-15T14:14:06.945+0000] {process_utils.py:187} INFO - ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
[2023-04-15T14:14:06.946+0000] {process_utils.py:187} INFO - hvac 1.1.0 requires requests<3.0.0,>=2.27.1, but you have requests 2.25.1 which is incompatible.
[2023-04-15T14:14:06.947+0000] {process_utils.py:187} INFO - docker 6.0.1 requires requests>=2.26.0, but you have requests 2.25.1 which is incompatible.
[2023-04-15T14:14:06.947+0000] {process_utils.py:187} INFO - apache-***-providers-http 4.2.0 requires requests>=2.26.0, but you have requests 2.25.1 which is incompatible.
[2023-04-15T14:14:06.948+0000] {process_utils.py:187} INFO - Successfully installed idna-2.10 requests-2.25.1
[2023-04-15T14:14:07.280+0000] {process_utils.py:187} INFO - 
[2023-04-15T14:14:07.280+0000] {process_utils.py:187} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.1
[2023-04-15T14:14:07.281+0000] {process_utils.py:187} INFO - [notice] To update, run: python -m pip install --upgrade pip
[2023-04-15T14:14:07.496+0000] {process_utils.py:179} INFO - Executing cmd: /tmp/venvzleimtqv/bin/python /tmp/venvzleimtqv/script.py /tmp/venvzleimtqv/script.in /tmp/venvzleimtqv/script.out /tmp/venvzleimtqv/string_args.txt
[2023-04-15T14:14:07.507+0000] {process_utils.py:183} INFO - Output:
